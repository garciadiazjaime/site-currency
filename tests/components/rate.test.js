import React from 'react'
import { shallow } from 'enzyme'

import Rate from '../../components/rate'

describe('Rate', () => {
  describe('main', () => {
    it('matches snapshot', () => {
      const wrapper = shallow(<Rate />)
  
      expect(wrapper).toMatchSnapshot()
    })

    it('renders props when passed', () => {
      const currency = 'currency'
      const rate = 'rate'
      const wrapper = shallow(<Rate currency={currency} rate={rate} />)
  
      expect(wrapper).toMatchSnapshot()
    })
  })
})
