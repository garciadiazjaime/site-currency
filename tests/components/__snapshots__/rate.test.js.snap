// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Rate main matches snapshot 1`] = `
<div>
  : 
</div>
`;

exports[`Rate main renders props when passed 1`] = `
<div>
  currency
  : 
  rate
</div>
`;
